.env.example

```diff
@@ -1 +1 @@
-REACT_APP_HOSPITALRUN_API=http://0.0.0.0:3001
\ No newline at end of file
+REACT_APP_HOSPITALRUN_API=http://0.0.0.0:5984
```

Comment:
Any reason to change the port? 5984 is usually what pouchdb runs on. We want to point to our server (which proxies to pouchdb).

Reply:
In the connecting to the server section I misunderstood step 4. Change the REACT_APP_HOSPITALRUN_API variable to point to the HospitalRun Development Server and had trouble running it. I got it working but forgot to switch it back sorry about that.

Reply:
No worries :)could you switch it back? Then we can merge this PR!

Reply:
I changed it back, let me know if there is any other changes that need to be made! Thanks!